Userland/Libraries/LibWeb/HTML/Window.h

```diff
@@ -269,6 +270,7 @@ class Window final
     JS::GCPtr<CSS::Screen> m_screen;
     JS::GCPtr<Navigator> m_navigator;
     JS::GCPtr<Location> m_location;
+    JS::GCPtr<CloseWatcherManager> m_close_watcher_manager;
```

Comment:
This needs to be marked in `Window::visit_edges()`, or the GC will nab it from us :)

Reply:
Ah I knew there was something I'd forgotten!